packages/oscal-react-library/src/components/OSCALCatalogBaseline.tsx

```diff
@@ -466,7 +420,11 @@ export default function OSCALCatalogBaseline() {
   let contact: ContactInfo = {};
   address = authorAddress;
   const createdModel = newOSCALModel === undefined ? Data : newOSCALModel;
-  const LabelText = openCatalogBaseline ? createdModel.title : " Catalogs & Baselines";
+  const LabelText = openCatalogBaseline
+    ? createdModel.title
+    : uploadNewCatalogBaseline
+    ? " Upload a Catalog or Baseline "
+    : " Catalogs & Baselines";
```

Comment:
I'm having a little difficulties reading this. To keep the code clear and concise, I would suggest breaking this into two constants, each handling one of the ternary operations.

Reply:
@Bronstrom Thanks for the insight, I broke it down to ternary operations.